HttpFs.IntegrationTests/Tests.fs

```diff
@@ -117,7 +117,8 @@ let recorded =
       Expect.stringContains (req.Value |> getHeader "accept-language") "en-GB" "accept-language should be set"
       Expect.stringContains (req.Value |> getHeader "accept-language") "en-US" "accept-language should be set"
       Expect.equal (req.Value |> getHeader "authorization") "QWxhZGRpbjpvcGVuIHNlc2FtZQ==" "authorization should be equal"
-      Expect.stringContains (req.Value |> getHeader "connection") "conn1" "connection should be set"
+      // difference in behaviour between .NET/Core and mono. mono repos connection as "close"
```

Comment:
...and .net core?

Reply:
Full .NET and Core have the same behavior. Mono seems to set the header to "close". I will play with it some more. It's either a difference in the underlying client behavior or the HTTP server (Suave).